src/aiconfigurator/generator/backends/trtllm.py

```diff
@@ -185,6 +188,10 @@ def generate(self, ctx: GeneratorContext) -> ArtifactBundle:
 
             pre_yaml = self._get_yaml_tpl(engine_tpl, {**pre_cfg, "dynamo_config": {**global_args, **worker_args["prefill"]}})
             dec_yaml = self._get_yaml_tpl(engine_tpl, {**dec_cfg, "dynamo_config": {**global_args, **worker_args["decode"]}})
+            pre_yaml_str = yaml.safe_dump(pre_yaml, sort_keys=False)
+            dec_yaml_str = yaml.safe_dump(dec_yaml, sort_keys=False)
```

Comment:
Do you know why agg calls set with `--cache_transceiver_backend default` will have 
```
cache_transceiver_config:
  backend: default
```
properly in its config, but it's missing in the disagg configs?